action.yml

```diff
@@ -46,30 +50,36 @@ runs:
       name: URL encode project name
       run: echo "::set-output name=project::${{ inputs.project }}" | sed -e 's:/:%2F:g'
       shell: bash
+      working-directory: ${{ inputs.working_directory }}
```

Comment:
it doesn't hurt, but this is unnecessary.  i'd just omit it.

Reply:
Fixed, thanks.